app/src/main/java/com/UserDatabase/UserRecord.java

```diff
@@ -0,0 +1,11 @@
+package com.UserDatabase;
+
+public class UserRecord {
+    public String username;
+    public float[] vector;
+

```

Comment:
we want to have average value of user vector, so we need to store number of vectors already calculated in this average

Reply:
I will add average vectors in separate commit

Reply:
+1